go/cmd/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	internal "github.com/Canhassi12/scylla-cloud-getting-started/Internal/commands"
+	"github.com/Canhassi12/scylla-cloud-getting-started/Internal/database"
+	"github.com/joho/godotenv"
+	"github.com/scylladb/gocqlx/v2"
+)
+
+func main() {
+	godotenv.Load(".env")
+	
+	session, err := database.New().Connect(); if err != nil {
+		println("error database connection =>", err.Error())
+		return
+	}
+
+	err = executeMigrateFile(session, "./Internal/database/migrations/migrate.cql"); if err != nil {
```

Comment:
Oh, no... don't do it. Never execute the migration when you are starting the application. Migration is not part of the application startup process, but the application configuration, it should never be executed without the user's agreement. Create a separate CLI to run the migrations/

Reply:
Is the same here.